lldb/tools/lldb-dap/Handler/InitializeRequestHandler.cpp

```diff
@@ -137,27 +137,30 @@ static void EventThreadFunction(DAP &dap) {
               lldb::SBBreakpoint::GetBreakpointEventTypeFromEvent(event);
           auto bp = Breakpoint(
               dap, lldb::SBBreakpoint::GetBreakpointFromEvent(event));
-          // If the breakpoint was originated from the IDE, it will have the
+          // If the breakpoint was originated from DAP, it will have the
```

Comment:
Is the first sentence redundant because we're in LLDB-DAP?

Reply:
No, it distinguishes from a breakpoint that was set through a command in the console or for example through your `~/.lldbinit`.